wger/weight/templates/weight_overview.html

```diff
@@ -38,6 +38,24 @@
 <div id="weight_diagram"></div>
 
 {% if is_owner %}
+{% if last_five_weight_entries_details %} 
+<table border="1" style="width:100%">
```

Comment:
better do here ``<table class="table">``, we can use the [bootstrap](http://getbootstrap.com/) CSS classes